pwndbg/gdblib/shellcode.py

```diff
@@ -13,6 +13,11 @@
 import pwnlib.shellcraft
 
 import pwndbg
+import pwndbg.gdblib.arch
+import pwndbg.gdblib.memory
+import pwndbg.gdblib.prompt
+import pwndbg.gdblib.regs
+import pwndbg.gdblib.vmmap
```

Comment:
I'm wondering why it was necessary to add these?

Reply:
To placate `mypy`, really.